src/main/java/ru/test/restservice/entity/LogEntry.java

```diff
@@ -0,0 +1,35 @@
+package ru.test.restservice.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class LogEntry {
+
+    @Id
+    @GeneratedValue
+    final private UUID id = null;
+
+    @Column(nullable = false)
+    public UUID user_id;
+
+    public UUID project_id;
+
+    @Column(nullable = false)
+    public LocalDateTime dateTime;
+
+    @Column(nullable = false)
+    public String action;
```

Comment:
Имхо стоит сделать `enum UserAction`, и в БД добавить колонку с описанием, если оно нужно
```suggestion
    public UserAction action;
```

Reply:
Я запишу это и поменяю, как договорюсь с пархомом, что в логах будет